### PR TITLE
IDEMPIERE-870 Fixed Assets using non-existant context variable

### PR DESCRIPTION
**Pull request by CarlosRuiz-globalqss**
https://idempiere.atlassian.net/browse/IDEMPIERE-870

### CODE REVIEW

File: org.adempiere.base/src/org/compiere/model/MAssetAddition.java [hunk 1]
```diff
@@ -433,12 +433,9 @@ public void setSourceAmt(BigDecimal amt)
 	 */
 	public void setIsApproved()
 	{
-		if(!isProcessed())
+		if(!isProcessed() && MRole.getDefault().isCanApproveOwnDoc())
 		{
-			String str = Env.getContext(getCtx(), "#IsCanApproveOwnDoc");
```
Comment (hengsin) [inline org.adempiere.base/src/org/compiere/model/MAssetAddition.java:438]: Perhaps we should add this variable instead, could be useful ?

Comment (mschnbeck) [reply org.adempiere.base/src/org/compiere/model/MAssetAddition.java:438]: I think so, too. I may be useful in other circumstances, too.

Comment (CarlosRuiz-globalqss) [reply org.adempiere.base/src/org/compiere/model/MAssetAddition.java:439]: Done, implemented in last commit

**Review by hengsin — APPROVED** [review]
looks good.

**Pull request merged**